src/components/molecules/AnnouncementMessage/AnnouncementMessage.scss

```
@@ -1,4 +1,5 @@
 @import "scss/constants.scss";
+$announcement-container-padding: 15px 0px;
```

[nit]
```suggestion

$announcement-container-padding: 15px 0px;
```

---

I would probably also make this just `15px` here in the variable, and maybe just call it `$spacing` or something else generic/short like that. Then we can reuse it in all of the 'spacing' related places within this file that need to be the same space.

```suggestion

$spacing: 15px;
```